_posts/2020-03-24-guide-for-unmaintained-open-source-projects.markdown

```diff
@@ -0,0 +1,192 @@
+yet when it comes to dealing with a possibly abandoned, unmaintained project, 
+there is no definitive guide for users, contributors, or maintainers.
+
+I hope that this can be a useful guide for our community.
+
+## Problem
+
+When do you declare that an open source project has been abandoned? How many 
+days have to go by until you start maintaining your own fork? What's the 
+standard for communicating with maintainers, contributors, and users? How do 
+you avoid `n` competing OSS forks of popular projects? How do you avoid 
+duplicated work by people who want to maintain popular, but unmaintained OSS 
+projects? What's the best way to find **that one fork** everybody is using?
+
+<!--more-->
+
+### Nights and Weekends 
+
+Too many applications out there are using libraries that depend on people 
+having enough time during nights and weekends to work on their side projects. 
+Most of the maintainers out there are not paid for their work. They might have 
+_some help_ from their employers, [Tidelift](https://tidelift.com), 
+[GitHub Sponsorships](https://github.com/sponsors), [Gitcoin](https://gitcoin.co), 
+or [OpenCollective](https://opencollective.com), but most projects don't have 
+financial support.
+
+No wonder many OSS maintainers burn out after years maintaining their project. 
+What happens when a maintainer burns out, starts using a different tech stack, 
+or even worse, dies?
+
+As users, we should be entirely grateful for their hard work, but we should also
+find a way to continue maintaining their OSS project, by starting a new fork 
+or backing an existing one.
+
+## Proposal 
+
+Depending on your role in the project, here are some strategies that could help
+users and contributors, or maintainers: 
+
+- Continuing maintenance for an abandoned OSS project
+- Finding someone else to maintain your OSS project
+
+### Why
+
+There is no clear way to continuing maintenance of an abandoned OSS project. 
+There are definitely great guides that share a few important concepts, but not 
+a lot of action steps for the case when an OSS project ends in 
+_maintenance limbo_: 
+
+- [https://opensource.guide/best-practices](https://opensource.guide/best-practices)
+- [https://www.linuxfoundation.org/resources/open-source-guides/winding-down-an-open-source-project/](https://www.linuxfoundation.org/resources/open-source-guides/winding-down-an-open-source-project/)
+- [https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951](https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951)
+
+### For OSS Users and Contributors
+
+What can you do as someone who actively uses or contributes to the OSS project? 
+For example: What can you do when you **really really** need to have support for 
+Rails 6.0 for your application but this library is blocking you?
+
+#### Best Judgement for Assessing Abandonment 
```

Comment:
This level isn't make visually clear that it's a title. Maybe if we add an extra line after the title or change the CSS would be much better

Reply:
@cleicar I created a story related to this: https://www.pivotaltracker.com/story/show/171987062